internal/network/command_private_link_attachment_connection.go

```diff
@@ -0,0 +1,64 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	networkingprivatelinkv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking-privatelink/v1"
+
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type privateLinkAttachmentConnectionOut struct {
+	Id                        string `human:"ID" serialized:"id"`
+	Name                      string `human:"Name" serialized:"name"`
+	Cloud                     string `human:"Cloud" serialized:"cloud"`
+	PrivateLinkAttachmentId   string `human:"Private Link Attachment ID" serialized:"private_link_attachment_id"`
+	Phase                     string `human:"Phase" serialized:"phase"`
+	AwsVpcEndpointId          string `human:"AWS VPC Endpoint ID,omitempty" serialized:"aws_vpc_endpoint_id,omitempty"`
```

Comment:
I see that `vpc_endpoint_id` is an attribute in both `status` and `spec` objects - [Link](https://docs.confluent.io/cloud/current/api.html#tag/Private-Link-Attachment-Connections-(networkingv1)/The-Private-Link-Attachment-Connections-Model)

@kabraham13 do these refer to different cloud resources?

Reply:
I see that they refer to the same cloud resource when I test it on devel but let's confirm with @kabraham13

Reply:
It looks like it is the same. I am not sure why. Maybe @arunvijayk has more context to why its in both spots

Reply:
I took a look at the `ToPrivateLinkAttachmentConnectionV2Proto` function in our repo, and the `VpcEndpointId` in `spec` and `status` are referring to the same `VpcEndpointId`

Reply:
Spec represents input from the customer - We need the customer to specify a VPC Endpoint ID while creating the PLATTC

Status represents information that we provide to the customer - The same vpcEndpointID that was specified by the customer when they created the PLATTC is returned here